site/docs/4.3/assets/js/src/application.js

```diff
@@ -10,19 +10,23 @@
  * For details, see https://creativecommons.org/licenses/by/3.0/.
  */
 
-/* global ClipboardJS: false, anchors: false, bsCustomFileInput: false */
+/* global ClipboardJS: false, anchors: false, bootstrap: false, bsCustomFileInput: false */
 
-(function ($) {
+(function () {
   'use strict'
 
-  $(function () {
+  document.addEventListener('DOMContentLoaded', function () {
```

Comment:
Might be worth double checking if we really need to wait until `DOMContentLoaded`.